src/property.html

```diff
@@ -0,0 +1,1066 @@
+                      >
+                        Read More
+                      </a>
+                    </p>
+                  </div>
+                </div>
+                <!-- pricing -->
+                <div class="flex gap-4 items-center sm:gap-0 justify-between">
+                  <div>
+                    <p class="font-medium text-sm text-color-addition-info">
+                      Price
+                    </p>
+                    <p class="font-semibold text-sm sm:text-lg text-color">
+                      $350,000
+                    </p>
+                  </div>
+                  <div
+                    class="px-5 py-[14px] rounded-lg bg-for-second-btn font-medium text-[10px] sm:text-sm text-color xl:w-60 text-center"
+                  >
+                    View Property Details
+                  </div>
+                </div>
+              </div>
+            </div>
+            <!-- buttons -->
+            <div>
+              <hr class="pb-2 color md:block hidden" />
+              <div class="xl:flex justify-between items-start">
+                <div class="hidden xl:inline-block">
+                  <p
+                    class="text-lg sm:text-xl font-medium text-color-addition-info"
+                  >
+                    <span class="text-color">01</span> of 10
+                  </p>
+                </div>
+                <div class="flex gap-2 justify-between items-center">
+                  <img
+                    src="/public/assets/propertypage/left.svg"
+                    alt="left-button"
+                    class="w-11 h-11 border-button p-3.5 rounded-full"
+                  />
+                  <p
+                    class="text-sm sm:text-xl font-medium text-color-addition-info xl:hidden"
+                  >
+                    <span class="text-color">01</span> of 60
+                  </p>
+                  <img
+                    src="/public/assets/propertypage/right.svg"
+                    alt="right-button"
+                    class="w-11 h-11 border-button p-3.5 bg-for-logo-section rounded-full"
+                  />
+                </div>
+              </div>
+            </div>
+          </div>
+        </section>
+        <!-- section personal info details -->
+        <section class="px-3 sm:px-8 xl:px-10 2xl:px-20">
+          <img
+            src="/public/assets/propertypage/pricing-logo-star.png"
+            alt="logo"
+          />
+          <div class="mt-1 ml-2 flex flex-col gap-10 lg:gap-12 2xl:gap-20">
+            <!-- heading -->
+            <div class="grid lg:grid-cols-6 gap-2">
+              <p
+                class="font-semibold lg:col-span-6 text-2xl xl:text-4xl text-color"
+              >
+                Let's Make it Happen
+              </p>
+              <p
+                class="font-medium lg:col-span-4 text-sm sm:text-base 2xl:text-lg text-color-addition-info"
```

Comment:
here as well

Reply:
this name used every where in html files

Reply:
use vs code searcher and replace it everywhere? Why arguing?

Reply:
ok

Reply:
i change it